pkg/cli/dump_test.go

```diff
@@ -90,6 +97,7 @@ func TestDumpRow(t *testing.T) {
 
 	c.RunWithArgs([]string{"sql", "-e", create})
 
+	c.sleepOffset()
```

Comment:
It seems like it'd be nicer to move this into a dump-specific helper for RunWithCapture/RunWithCaptureArgs, but feel free to disagree